v21.1/pg-catalog.md

```diff
@@ -0,0 +1,166 @@
+---
+title: pg_catalog
+summary: The pg_catalog schema contains read-only views that you can use for introspection into your database.
+toc: true
+---
+
+For PostgreSQL compatibility, CockroachDB includes a [virtual schema](virtual-schemas.html) called `pg_catalog`. The tables in the `pg_catalog` schema roughly correspond to the [system catalogs in PostgreSQL](https://www.postgresql.org/docs/13/catalogs-overview.html). `pg_catalog` tables are read-only.
+
+## Data exposed by `pg_catalog`
+
+The tables in `pg_catalog` correspond to a subset of the virtual tables and views that make up the PostgreSQL system catalogs. Not all PostgreSQL system catalogs have a corresponding table in `pg_catalog`, and not all `pg_catalog` tables have been fully implemented. See the table below for a detailed comparison between `pg_catalog` tables and the PostgreSQL 13 system catalogs.
+
+PostgreSQL 13 system catalog | `pg_catalog` table  
+-----------------------------|--------------
+`pg_aggregate` | `pg_aggregate`
+`pg_am` | `pg_am`
+`pg_amop` | `pg_amop` (unimplemented)
```

Comment:
LGTM but piggybacking on Vy's comment, I think we should clarify that unimplemented means that the table exists but the rows are not populated.